pkgs/by-name/wa/waybar/package.nix

```diff
@@ -1,71 +1,72 @@
-{ lib
-, stdenv
-, bash
-, fetchFromGitHub
-, fetchFromGitLab
-, SDL2
-, alsa-lib
-, catch2_3
-, fftw
-, glib
-, gobject-introspection
-, gtk-layer-shell
-, gtkmm3
-, howard-hinnant-date
-, hyprland
-, iniparser
-, jsoncpp
-, libdbusmenu-gtk3
-, libevdev
-, libinotify-kqueue
-, libinput
-, libjack2
-, libmpdclient
-, libnl
-, libpulseaudio
-, libsigcxx
-, libxkbcommon
-, meson
-, ncurses
-, ninja
-, pipewire
-, pkg-config
-, playerctl
-, portaudio
-, python3
-, scdoc
-, sndio
-, spdlog
-, sway
-, udev
-, upower
-, wayland
-, wayland-scanner
-, wireplumber
-, wrapGAppsHook3
+{
+  lib,
+  stdenv,
+  bash,
+  fetchFromGitHub,
+  fetchFromGitLab,
```

Comment:
This should be removed since it isn't being used.

Reply:
Done.